jax/_src/lax/lax.py

```diff
@@ -1222,7 +1232,10 @@ def scatter_mul(
   <https://www.tensorflow.org/xla/operation_semantics#scatter>`_, where
   multiplication is used to combine updates and values from `operand`.
 
-  The semantics of scatter are complicated and its API is subject to change.
+  The semantics of scatter are complicated, and its API might change in the
+  future. For most use cases, you should prefer the
+  :attr:`jaxlib.xla_extension.DeviceArray.at` property on JAX arrays which uses
```

Comment:
Elsewhere we have linked to ``` :attr:`jax.numpy.ndarray.at` ```. I think either will work (it's https://jax.readthedocs.io/en/latest/jax.numpy.html#jaxlib.xla_extension.DeviceArray.at vs. https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.ndarray.at.html#jax.numpy.ndarray.at) But let's choose one to standardize to.

Same comment elsewhere.

Reply:
Changed to `jax.numpy.ndarray.at`.